Formula/ffmpeg@2.8.rb

```diff
@@ -48,7 +49,6 @@ class FfmpegAT28 < Formula
   depends_on "libvo-aacenc" => :recommended
   depends_on "xvid" => :recommended
 
-  depends_on "faac" => :optional
```

Comment:
faac support was removed from FFmpeg 3.2. As this formula is for FFmpeg 2.8, there's no point in removing faac support.